### PR TITLE
DAM: New Footer

### PR DESCRIPTION
**Pull request by thomasdax98**
- new Footer design

Screenshots:

Selection Footer:
Bildschirm­foto 2023-02-28 um 12 51 35

Upload Footer:
Bildschirm­foto 2023-02-28 um 13 01 56 (2)

### CODE REVIEW

File: packages/admin/cms-admin/src/dam/DataGrid/footer/SelectionFooter.tsx [hunk 1]
```diff
@@ -146,3 +213,60 @@ export const DamSelectionFooter: React.VoidFunctionComponent<DamSelectionFooterP
...
+
+        if (result) {
+            if (result.errors) {
+                setInternalHasErrors(true);
+                setTimeout(() => {
+                    setInternalHasErrors(false);
+                }, 3000);
+            } else {
+                await apolloClient.refetchQueries({ include: [namedOperations.Query.DamItemsList] });
+                clearDamItemCache(apolloClient.cache);
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/dam/DataGrid/footer/SelectionFooter.tsx:258]: You first refetch queries and then clear the cache? Wouldn't this throw away the fetched queries?

File: packages/admin/cms-admin/src/dam/DataGrid/footer/SelectionFooter.tsx [hunk 2]
```diff
@@ -146,3 +213,60 @@ export const DamSelectionFooter: React.VoidFunctionComponent<DamSelectionFooterP
...
+interface IconButtonProps {
+    title: NonNullable<React.ReactNode>;
+    onClick?: () => void;
+    executeMutation?: () => Promise<{ errors: readonly GraphQLError[] | undefined } | undefined>;
+    icon: React.ReactNode;
+    loading?: boolean;
+    hasErrors?: boolean;
+}
+
+const IconButton = ({ title, onClick, executeMutation, icon, loading: externalLoading, hasErrors: externalHasErrors }: IconButtonProps) => {
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/dam/DataGrid/footer/SelectionFooter.tsx:232]: I'd call this FooterAction or FooterActionButton

File: packages/admin/cms-admin/src/dam/DataGrid/footer/SelectionFooter.tsx (hunk 1 above)
Comment (thomasdax98) [reply packages/admin/cms-admin/src/dam/DataGrid/footer/SelectionFooter.tsx]: Yes, you are right. The refetch is not necessary

File: packages/admin/cms-admin/src/dam/DataGrid/footer/DamFooter.tsx [hunk 3]
```diff
@@ -9,21 +9,20 @@ const FooterBar = styled(Paper)`
     left: 50%;
     transform: translateX(-50%);
 
-    min-width: 1280px;
+    min-width: 321px;
```
[the remark this answers is not in the document]
Comment (thomasdax98) [reply packages/admin/cms-admin/src/dam/DataGrid/footer/DamFooter.tsx]: I took it from the screen design. But yes, it's somewhat arbitrary. I removed the min-width and increased the gap instead

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**